chore: disable start button and add tooltip for subscriptions

subirjolly opened the pull request:
Closes: https://github.com/influxdata/data-acquisition/issues/284

## BEFORE
![BEFORE](https://user-images.githubusercontent.com/1637395/169608008-adac514c-7bea-4a42-a10a-b10a1a831a02.gif)

## AFTER
![AFTER](https://user-images.githubusercontent.com/1637395/169608032-76078731-9161-40c4-ab50-8bde486554a1.gif)

helenosheaa reviewed (approved):
Change looks good!

I've tried rerunning your tests on ci but it's failing on:
```
CypressError: `cy.request()` failed on:

https://twodotoh-ci6hqoupuz53h62tsp.a.influxcloud.dev.local/debug/flush

The response we received from your web server was:

  > 404: Not Found
```

You may need to rebase. It'd also be good to add a e2e test for this change after this is merged in a separate PR.

subirjolly commented:
> Change looks good!
> 
> I've tried rerunning your tests on ci but it's failing on:
> 
> ```
> CypressError: `cy.request()` failed on:
> 
> https://twodotoh-ci6hqoupuz53h62tsp.a.influxcloud.dev.local/debug/flush
> 
> The response we received from your web server was:
> 
>   > 404: Not Found
> ```
> 
> You may need to rebase. It'd also be good to add a e2e test for this change after this is merged in a separate PR.

Added a test and fixed the bug which was preventing clicks on Start/Stop button
